doc/TuningCycle.md

```diff
@@ -26,3 +26,80 @@ please ask for them. The Tuna team does not provide these.
 phase of all the recurrent configurations takes up to a few days. There are many configurations
 used for each network and one should try and use as many machines as possible to speed up
 the tuning part. The current configurations we tune for are:
+
+## Tuning Steps
+
+### Add Network Configurations 
+The config table contains network configurations. If provided with a text file of MIOpenDriver
+commands, the import script can translate those commands and populate the config table. 
+Additionally the user may provide a name to tag a configuration for easier recall later.
+Tags are stored in the config_tags table.
+
+./import_configs.py -t resnet50 -f ../utils/recurrent_cfgs/resnet50.txt
+-t - tag
+-f - filepath
+
+### Add Solvers
+The solver table contains MIOpen solvers and solver characteristics. This should be initialized
+and updated when an MIOpen version adds new solvers.
+
+./go_fish.py --local_machine --update_solvers
+--local_machine - run directly on this machine
+
+### Create a Tuning session
+Session will track the architecture and skew, as well as the miopen version and 
+rocm version of the tuning session.
+
+This command will need to be run from inside the tuning environment eg MITunaX docker
+and will populate the table with the version and architecture information.
+
+./go_fish.py --local_machine --init_session -l reason
+--init_session - create a session entry
+-l             - reference text description
+
+### Create Applicability
+Each network configuration has a set of applicable solvers. In this step we update the
+solver_applicability table with our applicable solvers for each configuration for the session.
+
+./go_fish.py --local_machine --update_applicability --session_id 1
```

Comment:
Add every command in quotations so it shows up nicely formatted and distinct from text.

```suggestion
```./go_fish.py --local_machine --update_applicability --session_id 1```
```